topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md

```diff
@@ -0,0 +1,842 @@
+>
+>
+{: .comment}
+
+The main reason underlying the greater accuracy of StringTie most likely derives from its optimization criteria. By balancing the coverage (or flow) of each transcript across each assembly, it incorporates depth of coverage constraints into the assembly algorithm itself. When assembling a whole genome, coverage is a crucial parameter that must be used to constrain the algorithm; otherwise an assembler may incorrectly collapse repetitive sequences. Similarly, when assembling a transcript, each exon within an isoform should have similar coverage, and ignoring this parameter may produce sets of transcripts that are parsimonious but wrong ({% cite Pertea2015 %}).
+
+
+> <details-title>StringTie long RNA-seq assembly</details-title>
+>
+> To handle the high error rates in the long reads, StringTie implements two techniques. First, it correct potentially wrong splice sites by checking all the splice sites present in the alignment of a read with a high-error alignment rate. If a splice site is not supported by any low-error alignment reads, then it tries to find a nearby splice site (within 10 bp, by default) that is supported by the most alignments among all nearby splice sites. Secondly, it implements a pruning algorithm that reduces the size of the splicing graph to a more realistic size by removing the edges in the graph starting from the edge least supported by reads to the most supported edge, until the number of nodes in the splicing graph falls under a given threshold (by default 1000 nodes).
+>
+{: .details}
+
+Despite in this training we make use of RNA STAR as mapping tool, it is possible to use different splice-aware aligner such as HISAT2. Independently of the tool, each record with a spliced alignment should have the XS tag in the SAM/BAM file, which indicates the genomic strand from which the RNA that produced the read originated . 
+
+> <hands-on-title> Task description </hands-on-title>
+>
+> 1. {% tool [StringTie](toolshed.g2.bx.psu.edu/repos/iuc/stringtie/stringtie/2.2.1+galaxy1) %} with the following parameters:
+>    - *"Input options"*: `Short reads`
+>        - {% icon param-file %} *"Input short mapped reads"*: `Mapped health collection`
+>    - *"Use a reference file to guide assembly?"*: `Use reference GTF/GFF3`
+>        - *"Reference file"*: `Use a file from history`
+>            - {% icon param-file %} *"GTF/GFF3 dataset to guide assembly"*: `gencode.v43.annotation.gtf.gz`
+>        - *"Use Reference transcripts only?"*: `Yes`
```

Comment:
Yes, you are right. I just decided to use StringTie because originally we tried also to identify new transcripts, and in a next review I would like to extend it.